src/components/task-edit.js

```diff
@@ -1,6 +1,106 @@
-export const createCardTaskEditTemplate = () => {
+import {COLORS, DAYS, MONTH_NAMES} from '../const.js';
+import {formatTime} from '../utils.js';
+
+// функция вывода хештегов
+const createHashtags = (tags) => {
+  return Array.from(tags)
+    .map((tag) => {
+      return (
+        `<span class="card__hashtag-inner">
+          <input
+            type="hidden"
+            name="hashtag"
+            value=${tag}
+            class="card__hashtag-hidden-input"
+          />
+          <p class="card__hashtag-name">
+            #${tag}
+          </p>
+          <button
+              type="button"
+              class="card__hashtag-delete"
+          >
+            delete
+          </button>
+        </span>`
+      );
+    })
+    .join(`\n`);
+};
+
+// отрисовываем маркеры с цветами и выделяем активной ту где совпадает цвет классом checked
+const createColorsMarkup = (colors, currentColor) => {
+  return colors
+    .map((color) => {
+      return (
+        `<input
+          type="radio"
+          id="color-${color}-4"
+          class="card__color-input card__color-input--${color} visually-hidden"
+          name="color"
+          value="${color}"
+          ${currentColor === color ? `checked` : ``}
+        />
+        <label
+          for="color-${color}-4"
+          class="card__color card__color--${color}"
+          >${color}</label
+        >`
+      );
+    })
+    .join(`\n`);
+};
+
+// Отрисовываем дни недели и отмечаем те что повторяются (имеют значение true) как checked
+const createRepeatingDaysMarkup = (days, repeatingDays) => {
+  return days
+    .map((day) => {
+      const isChecked = repeatingDays[day];
+      return (
+        `<input
+          class="visually-hidden card__repeat-day-input"
+          type="checkbox"
+          id="repeat-${day}-4"
+          name="repeat"
+          value="${day}"
+          ${isChecked ? `checked` : ``}
+        />
+        <label class="card__repeat-day" for="repeat-${day}-4"
+          >${day}</label
+        >`
+      );
+    })
+    .join(`\n`);
+};
+
+export const createCardTaskEditTemplate = (tasks) => {
```

Comment:
```suggestion
export const createCardTaskEditTemplate = (task) => {
```
Тут вроде одна задача

Reply:
fixed